src/services/countriesCount.js

```diff
@@ -0,0 +1,16 @@
+export function getCountriesCountFromLocalStorage() {
+  const myCountriesCount = JSON.parse(localStorage.getItem("count")) || [];
+  return myCountriesCount;
+}
+
+export function addCountryCountToLocalStorage() {
+  const myCountriesCount = getCountriesCountFromLocalStorage();
+  const newCount = Number(myCountriesCount) + 1;
+  localStorage.setItem("count", JSON.stringify(newCount));
+}
+
+export function removeCountryCountFromLocalStorage() {
+  const myCountriesCount = getCountriesCountFromLocalStorage();
+  const newCount = Number(myCountriesCount) - 1;
+  localStorage.setItem("count", JSON.stringify(newCount));
+}
```

Comment:
In our opinion the filename is a bit out of place. We would never expect localStorage functionality in here. You named your other files like `...MarkersStorage` what is way better.